src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1260,6 +1266,41 @@ private void cleanupChatMessages()
         }
     }
 
+    @Override
+    public void setCustomNameTag(final String name)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS3776)